handlers/product-move-api/cfn.yaml

```
@@ -305,13 +305,14 @@ Resources:
   RefundQueue:
     Type: AWS::SQS::Queue
     Properties:
-      VisibilityTimeout: 3000
+      VisibilityTimeout: 43200
+      MessageRetentionPeriod: 604800
```

is it possible to put a comment saying how many hours/days this is?
```suggestion
      VisibilityTimeout: 43200 # 12 hours
      MessageRetentionPeriod: 604800 # 7 days
```

---

I wonder if we should make it a few minutes longer than 7 days as well, just to make sure we get the final retry before the end of the message retention period

---

Yes, I have added the comment and also added 30 min to message retention period